OpenRA.Game/Map/MapCache.cs

```diff
@@ -122,6 +122,48 @@ public void LoadMaps()
 			}
 		}
 
+		public IEnumerable<Map> EnumerateMapsWithoutCaching(MapClassification classification = MapClassification.System)
```

Comment:
For this to work properly you will need to change the `MapClassification` definition to:
```csharp
// Used for grouping maps in the UI and for filtering
[Flags]
public enum MapClassification
{
	Unknown = 0,
	System = 1,
	User = 2,
	Remote = 4
}
```